pkg/agent/loadbalancer/loadbalancer_test.go

```diff
@@ -105,7 +105,7 @@ func TestFailOver(t *testing.T) {
 		DataDir:   tmpDir,
 	}
 
-	lb, err := New(cfg.DataDir, SupervisorServiceName, cfg.ServerURL)
+	lb, err := New(cfg.DataDir, SupervisorServiceName, cfg.ServerURL, 0)
```

Comment:
Can you define the `0` as a const with a descriptive name?

Reply:
This would be dependent on: https://github.com/k3s-io/k3s/pull/2772/files#r555299396